### PR TITLE
Add alphanumeric validator

### LINKED ISSUE #81: Add an alphanumeric validator
Currently there is no validator for alphanumeric (Characters and numbers only). It would be nice to have one for reuse.

### PR DESCRIPTION
**Pull request by colanconnon**
fixes #81

### CODE REVIEW

File: keg_elements/forms/validators.py [hunk 1]
```diff
@@ -4,6 +4,34 @@
...
+        A validator to make sure than a form field contains only alphanumeric data
+
+        usage example:
+            import keg_elements.forms.validators as validators
+
+            wtforms.StringField('AlphaNumeric', validators=[
+                            validators.ValidateAlphaNumeric()
+                        ])
+    """
+    regex = re.compile(r'^[a-zA-Z0-9][a-zA-Z0-9\s]*$')
```
Comment (tjlevel12) [inline keg_elements/forms/validators.py]: Do we need to use a regex here? Given the tests below, it seems like we could accomplish the same thing just by checking `field.data.isalnum()`.

One piece not handled by `isalnum()` are non-leading spaces, which are permitted by the regex (so `' abc'` fails, but `'abc '` is valid). If these non-leading spaces are necessary, they should probably be added to the tests.

Comment (bladams) [reply keg_elements/forms/validators.py]: `isalnum` will match unicode chars as well. I think this validator is meant to restrict to ascii. If not then the expression will need to be re-written anyway.

Comment (colanconnon) [reply keg_elements/forms/validators.py]: @tjlevel12 good catch on the white space issue, in this case I don't think we want to support them. @bladams what do you think?

File: keg_elements/forms/validators.py [hunk 2]
```diff
@@ -4,6 +4,34 @@
...
+        A validator to make sure than a form field contains only alphanumeric data
+
+        usage example:
+            import keg_elements.forms.validators as validators
+
+            wtforms.StringField('AlphaNumeric', validators=[
+                            validators.ValidateAlphaNumeric()
+                        ])
+    """
+    regex = re.compile(r'^[a-zA-Z0-9][a-zA-Z0-9]*$')
```
[the remark this answers is not in the document]
Comment (colanconnon) [reply keg_elements/forms/validators.py]: @tjlevel12 Yah I think you are right. Good feedback!

[the remark this answers is not in the document]
Comment (rsyring) [reply keg_elements/forms/validators.py]: If TJ's assessment of the intention is correct, I do think his regex is optimal.

**Comment by colanconnon** [comment]
@tjlevel12 updated!

**Pull request merged**